volatility3/framework/symbols/windows/__init__.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, *args, **kwargs) -> None:
         self.set_type_class("_KTHREAD", extensions.KTHREAD)
         self.set_type_class("_LIST_ENTRY", extensions.LIST_ENTRY)
         self.set_type_class("_EPROCESS", extensions.EPROCESS)
+        self.set_type_class("_ERESOURCE", extensions.ERESOURCE)
```

Comment:
Just to check, this is present in all windows' back through the history of time?  If there's versions where this symbol didn't exist in this form it should be optional like the calls below.

(Absolutely may be, I just check now because of breakages we've had in the past, perfectly happy keeping it here if we're confident this won't prevent analysis of really old/really new versions of windows)...

Reply:
It looks like it is as old as at least NT 3.51, and probably before: https://www.geoffchappell.com/studies/windows/km/ntoskrnl/api/ex/resource/eresource.htm